### DISCUSSION TITLE
maybe a bug?

### DISCUSSION THREAD

**Discussion by JSchmidt63** [alibaba/designable#376] [General]
1. use Ant Design
2. change to English
3. create a tab
4. create 2 panels
5. drop a input in the first pane

bug: see in the second pane showing the same input

{
  "form": {
    "labelCol": 6,
    "wrapperCol": 12
  },
  "schema": {
    "type": "object",
    "properties": {
      "ffpqapz14f4": {
        "type": "void",
        "x-component": "FormTab",
        "x-component-props": {},
        "x-designable-id": "ffpqapz14f4",
        "properties": {
          "s9mzpevna5z": {
            "type": "void",
            "x-component": "FormTab.TabPane",
            "x-component-props": {
              "tab": "Unnamed Title"
            },
            "x-designable-id": "s9mzpevna5z",
            "properties": {
              "dxcs94vr4tc": {
                "type": "string",
                "title": "Input",
                "x-decorator": "FormItem",
                "x-component": "Input",
                "x-validator": [],
                "x-component-props": {},
                "x-decorator-props": {},
                "x-designable-id": "dxcs94vr4tc",
                "x-index": 0
              }
            },
            "x-index": 0
          },
          "d5wsbg4pirx": {
            "type": "void",
            "x-component": "FormTab.TabPane",
            "x-component-props": {
              "tab": "Unnamed Title"
            },
            "x-designable-id": "d5wsbg4pirx",
            "x-index": 1
          }
        },
        "x-index": 0
      }
    },
    "x-designable-id": "qfdu5gjxg2s"
  }
}

Cheers

**Comment by ilyas-ascend** [comment]
This is due to the re-render issue 
Just change accordingly as mentioned in the image 
Path: formily\antd\src\components\FormTab\preview.tsx:58

![image](https://github.com/alibaba/designable/assets/103880962/2545f81e-2942-4614-abf7-79a66005ba0a)

**Reply by wxzhhh** [reply]
i think this is due to version of antd in template.ejs, this antd's version must same as antd's version in node_modules, otherwise antd's style will lose
![image](https://github.com/alibaba/designable/assets/114121254/a235b264-a644-4a90-bee1-0414307aa01b)
![image](https://github.com/alibaba/designable/assets/114121254/62c3bccf-22e4-4756-90c6-06763d7f9bb1)
